Sources/TuistSigning/ProvisioningProfile/ProvisioningProfileParser.swift

```diff
@@ -44,6 +44,11 @@ final class ProvisioningProfileParser: ProvisioningProfileParsing {
         let plistData = Data(unencryptedProvisioningProfile.utf8)
         let provisioningProfileContent = try PropertyListDecoder().decode(ProvisioningProfile.Content.self, from: plistData)
 
+        let developerCertificateFingerprints = try provisioningProfileContent.developerCertificates.map { (data) throws -> String in
+            let certificateParser = CertificateParser()
```

Comment:
this is not properly injected - you should use only the protocol interface in the code itself (and init the default implementation in the init of this class)

Reply:
The `map` can be rewritten in a more concise version `.developerCertificates.map(certificateParser.parseFingerPrint)` as I think `(data) throws -> String in` is unnecessarily verbose in this case